src/terminal/parser/stateMachine.cpp

```diff
@@ -367,7 +368,7 @@ static constexpr bool _isDcsPassThroughValid(const wchar_t wch) noexcept
 // - True if it is. False if it isn't.
 static constexpr bool _isActionableFromGround(const wchar_t wch) noexcept
 {
-    return (wch <= AsciiChars::US) || _isC1Csi(wch) || _isC1Dcs(wch) || _isDelete(wch);
+    return (wch <= AsciiChars::US) || _isC1ControlCharacter(wch) || _isDelete(wch);

```

Comment:
I just realized by doing C1 SS3(8/15) and C1 OSC(9/13) are also supported. And why are they not supported before? Is there anything special about them?